modules/virt-about-deploying-templates-custom-namespace.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-about-deploying-templates-custom-namespace.adoc
+
+[id="virt-about-deploying-templates-custom-namespace_{context}"]
+= About deploying templates to a custom namespace
+
+To add templates to a custom namespace, edit the `hco` custom resource (CR) and specify a custom namespace. Then, add the Red Hat templates to that custom namespace to enable the `ssp-operator` to populate the templates in the custom namespace.
+
+[NOTE]
+====
+When you create a private namespace as a cluster administrator, you can assign roles with permissions to enable other users to access that namespace and deploy templates to that namespace.
```

Comment:
* Remove all usages of `private`
* Custom namespace should have role/rolebinding for ssp-operator to ensure it can manage the templates.
* Any templates not found in openshift namespace will not be displayed in UI during VM creation.

Reply:
Updated the content to reflect this feedback.